code/datums/ruins/lavaland.dm

```diff
@@ -0,0 +1,87 @@
+/datum/map_template/ruin/lavaland
+	prefix = "_maps/map_files/RandomRuins/LavaRuins/"
+
+/datum/map_template/ruin/lavaland/biodome
+	cost = 5
+	allow_duplicates = FALSE
+
+/datum/map_template/ruin/lavaland/biodome/beach
```

Comment:
May want to re-order these, to the best of your ability, as TG has them. If someone ever wants to port/implement the rest of them, it'll be a lot easier, at a glance, to tell which ones are/aren't implemented yet.